recipes/nbexamples/meta.yaml

```diff
@@ -9,7 +9,7 @@ package:
 source:
   fn: {{ gh_repo }}-v{{ version }}.tar.gz
   url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/v{{ version }}.tar.gz
-  sha256: 96926c36333e57aee3820722f1107fbf6a3df9113b8585b14e572fccce385923
+  sha256: f391b42ea6856d92a06fafa18ede464e400c9b2804a2a022d0b7197fd4bd4061
```

Comment:
I'm ok with the change and it does seem to be for the better as it is now `<name>-<tag>.<comp>` as opposed to `<tag>.<comp>`. Though I think it is a bit weird/not nice that they didn't bother to tell anyone about it.